### PR TITLE
More examples

### PR DESCRIPTION
**Pull request by rbaheti**
### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Checklist

- [x] I have documented all new methods.
- [ ] I have written tests for all new methods/functionality.
- [x] I have written examples for all new methods/functionality.

### Checklist of new examples

- [x] Geomap Point size
- [x] Geomap tiles and ocean configuration
- [x] Geomap fitFilter example
- [x] Geomap zoom configuration

### CODE REVIEW

File: example/coordinate-point-size.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+[height]: 550
+
+# Coordinate Point Size on a Map
```
Comment (davelandry) [inline example/coordinate-point-size.md]: Changing Coordinate Point Sizes

File: example/coordinate-point-size.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+For a coordinate point [Geomap](https://d3plus.org/docs/#Geomap), you can specify the point size by setting the [pointSize](https://d3plus.org/docs/#Geomap.pointSize) property to an indicator you want along with the [pointSizeMin](https://d3plus.org/docs/#Geomap.pointSizeMin) and [pointSizeMax](https://d3plus.org/docs/#Geomap.pointSizeMax) properties for specifying the range of the point size, as shown in the following example:
```
Comment (davelandry) [inline example/coordinate-point-size.md]:
```md
For a coordinate point [Geomap](https://d3plus.org/docs/#Geomap), you can enable dynamic point sizing by setting the [pointSize](https://d3plus.org/docs/#Geomap.pointSize) property to an accessor that returns the data value you want to use. Additionally, the minimum and maximum values used in the scaling can be overridden using the [pointSizeMin](https://d3plus.org/docs/#Geomap.pointSizeMin) and [pointSizeMax](https://d3plus.org/docs/#Geomap.pointSizeMax) methods:
```

File: example/remove-tiles-from-geomap.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+[height]: 550
+
+# Removing Background Tiles from a Geomap
+
+[Geomap](https://d3plus.org/docs/#Geomap) tiles can be removed by setting the `tiles` property to false. Similarly, by setting `ocean` property to transparent will hide ocean on the map. To hide the zoom button at the top of the map, you can set `zoom` property to false. Following example cover all three of these properties:
```
Comment (davelandry) [inline example/remove-tiles-from-geomap.md]:
```md
[Geomap](https://d3plus.org/docs/#Geomap) background tiles can be removed by setting the `tiles` property to `false`. The `ocean` property can then be changed to alter the overall background color, with `"transparent"` being used to remove the background color entirely.
```

File: example/remove-tiles-from-geomap.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+new d3plus.Geomap()
+  .config({
+    data: popData,
+    groupBy: "id",
+    colorScale: "population",
+    topojson: "https://d3plus.org/topojson/states.json",
+    fitFilter: function(d) {
+      return ["02", "15", "43", "60", "66", "69", "72", "78"].indexOf(d.id) < 0;
+    },
+    zoom: false,
```
Comment (davelandry) [inline example/remove-tiles-from-geomap.md]: make a different example to show the different zoom behaviors you can disable

File: example/remove-tiles-from-geomap.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+    ocean: "transparent",
+    tiles: false,
```
Comment (davelandry) [inline example/remove-tiles-from-geomap.md:19]: remove trailing comma

File: example/show-required-area.md [hunk 6]
```diff
@@ -0,0 +1,21 @@
+[height]: 550
+
+# Show Required Area on Geomap
```
Comment (davelandry) [inline example/show-required-area.md]: Change Default Geomap Zooming

File: example/show-required-area.md [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+
+Topojson files sometimes include small geographies that negatively impact how the library determines the default zoom level (for example, a small island or territory far off the coast that is barely visible to the eye). The fitFilter method can be used to remove specific geographies from the logic used to determine the zooming, in this case removing small islands, like Guam and American Samoa, and removing Alaska in order to focus on the 48 contiguous US states.
```
Comment (davelandry) [inline example/show-required-area.md]: add link for `fitFilter`

**Review by davelandry — CHANGES_REQUESTED** [review]

File: package.json [hunk 8]
```diff
@@ -30,7 +30,7 @@
     "d3-selection": "^1.2.0",
     "d3-tile": "0.0.4",
     "d3plus-common": "^0.6.36",
-    "d3plus-shape": "^0.14.6",
+    "d3plus-shape": "^0.14.11",
```
Comment (davelandry) [inline package.json]: why is this being committed to this PR?

File: src/Geomap.js [hunk 9]
```diff
@@ -58,8 +58,13 @@ export default class Geomap extends Viz {
 
     this._shape = constant("Circle");
     this._shapeConfig = assign(this._shapeConfig, {
+      ariaLabel: (d, i) => `${this._drawLabel(d, i)}, ${this._pointSize(d, i)}`,
```
Comment (davelandry) [inline src/Geomap.js]: why is this being committed to this PR?

**Review by davelandry — CHANGES_REQUESTED** [review]

File: package.json (hunk 8 above)
Comment (rbaheti) [reply package.json]: Sorry, reverted it.

File: src/Geomap.js (hunk 9 above)
Comment (rbaheti) [reply src/Geomap.js]: Sorry, reverted it.

File: example/disable-geomap-zoom-controls.md [hunk 10]
```diff
@@ -0,0 +1,22 @@
+[height]: 550
+
+# Disable Geomap Zoom Controls
```
Comment (davelandry) [inline example/disable-geomap-zoom-controls.md]: Disable Geomap Zooming

File: example/disable-geomap-zoom-controls.md [hunk 11]
```diff
@@ -0,0 +1,22 @@
...
+
+For a [Geomap](https://d3plus.org/docs/#Geomap), zoom controls are enabled by default. To disable them, set the [zoom](https://d3plus.org/docs/#Viz.zoom) property to `false`.
```
Comment (davelandry) [inline example/disable-geomap-zoom-controls.md]:
```md
For a [Geomap](https://d3plus.org/docs/#Geomap), zooming is enabled by default. To disable them entirely, set the [zoom](https://d3plus.org/docs/#Viz.zoom) property to `false`.
```

File: example/remove-tiles-from-geomap.md [hunk 12]
```diff
@@ -17,8 +17,7 @@ new d3plus.Geomap()
       return ["02", "15", "43", "60", "66", "69", "72", "78"].indexOf(d.id) < 0;
     },
     zoom: false,
-    ocean: "transparent",
-    tiles: false,
+    ocean: "transparent"
```
Comment (davelandry) [inline example/remove-tiles-from-geomap.md:20]: looks like you removed `tiles` instead of removing `zoom`

**Review by davelandry — CHANGES_REQUESTED** [review]

**Review by davelandry — CHANGES_REQUESTED** [review]
There are downgrading changes in `package.json`, `package-lock.json`, and `src/Geomap.js`. Try to resolve these by pulling/merging from master into this branch (not the other way around). If that doesn't work, please close this PR and make a new one based on the current master branch.

**Pull request closed without merging**